torchtune/models/qwen2/_model_builders.py

```diff
@@ -40,6 +40,52 @@ def qwen2_7b() -> Union[TransformerDecoder, TiedEmbeddingTransformerDecoder]:
     )
 
 
+def qwen2_0_5b() -> Union[TransformerDecoder, TiedEmbeddingTransformerDecoder]:
+    """
+    Builder for creating a Qwen2 model initialized w/ the default 0.5B parameter values
+    from https://huggingface.co/Qwen/Qwen2-0.5B-Instruct
+
+    Returns:
+        Union[TransformerDecoder, TiedEmbeddingTransformerDecoder]: Instantiation of Qwen2 0.5B model
+    """
+    return qwen2(
+        vocab_size=151936,
+        num_layers=24,
+        num_heads=14,
+        num_kv_heads=2,
+        embed_dim=896,
+        intermediate_dim=4864,
+        max_seq_len=32768,
+        attn_dropout=0.0,
+        norm_eps=1e-06,
+        rope_base=1000000.0,
+        tie_word_embeddings=True,
+    )
+
+
+def qwen2_1_5b() -> Union[TransformerDecoder, TiedEmbeddingTransformerDecoder]:
+    """
+    Builder for creating a Qwen2 model initialized w/ the default 1.5B parameter values
```

Comment:
Since `tied_embeddings=True`, we'll always be returning `TiedEmbeddingTransformerDecoder`, right?